packages/ui/src/components/Toasts/ToastContent.tsx

```diff
@@ -20,14 +20,26 @@ const ToastContent = ({ className, toast }: ToastContentProps) => {
   }, [link])
 
   return (
-    <div className={className}>
+    <div
+      className={className}
+      data-cy="notification-wrapper"
+    >
       <div className="iconContainer">
         {type === 'success' ? (
-          <HiOutlineCheckCircle size={25} />
+          <HiOutlineCheckCircle
+            data-cy="notification-icon-success"
+            size={25}
```

Comment:
It's a small thing for sure but the sizes are normally even, so 24 is probably a good size

Reply:
unrelated, but small enough that we can squeeze this in, pushed the change :+1: